### DISCUSSION TITLE
Einstellungen Pylontech

### DISCUSSION THREAD

**Discussion by TheMikeCAD** [hoylabs/OpenDTU-OnBattery#818] [General]
Hallo zusammen,
ich habe eine Pylontech US5000 - gerade im Testbetrieb mit folgenden Einstellungen:

Batterienutzung Start-Schwellwert 30%
Batterienutzung Stop-Schwellwert 15%
Full-Solar-Passthrough Start Schwellwert 90%

Batterienutzung Start-Schwellwert 51,75 V
Batterienutzung Stop-Schwellwert 48 V
Full-Solar-Passthrough Start-Schwellwert 52 V
Full-Solar-Passthrought Stop-Schwellwert 49V

Frage 1: Sind die Werte stimmig? Komme mit den Beschreibungen nicht wirklich klar.
Frage 2: Bei meinem Test ist der Akku jetzt in den Alarm gelaufen, weil das System die Ladung nicht beendet hat.
Ist das normal? bzw. wie kann ich das verhindern. Denke am Ende schaltet dann das BMS aus?!

Der Hoymiles (600W) bekommt die Leistung der Anlage nicht weg.. wie kann ich das sinnvoll einstellen?

Nachtrag: Hab die PV Leistung kurzerhand abgeklemmt.. jetzt ist die Akkuspannung wieder auf 50,79 V gefallen.. war vorher bei 53 V - wie gibs das denn??
Laut DTU ist das Laden der Batterie nicht möglich, aber lädt trotzdem mit einem Strom von 2,1A??

![Pylontech_DTU](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/149927355/e7b492a0-cc6f-4b5e-aa58-cf8562c4ab26)

Verstehe ich das richtig, dass die Schwellwerte in % immer analog der Werte in V sein müssen?

Vielen Dank und Gruß

**Comment by Manos1966** [comment]
Hallo @TheMikeCAD 

Wie wird denn deine Batterie geladen?
Welche Einstellungen hast du im Laderegler eingegeben?

Hast du auch hier ein Blick geworfen?
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Victron-MPPT-battery-charger
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Victron-MPPT-Tipps
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Pylontech-battery
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps

**Comment by spcqike** [comment]
> Frage 2: Bei meinem Test ist der Akku jetzt in den Alarm gelaufen, weil das System die Ladung nicht beendet hat.
Ist das normal? bzw. wie kann ich das verhindern. Denke am Ende schaltet dann das BMS aus?!

was hast du denn für ein laderegler? Dieser muss natürlich auch eingestellt werden, dass er bei der ladeschlussspannung stehen bleibt. 

Ja das BMS rettet irgendwann die Batterie. Aber damit trennt es diese ja komplett. Dann ist sie natürlich auch für den wechselrichter nicht verfügbar. 

> nachtrag:%20Hab%20die%20PV%20Leistung%20kurzerhand%20abgeklemmt..%20jetzt%20ist%20die%20Akkuspannung%20wieder%20auf%2050,79%20V%20gefallen..%20war%20vorher%20bei%2053%20V%20-%20wie%20gibs%20das%20denn??

das ist normal. Der laderegler liegt ja eine höhere Spannung an. Bzw. Die Zielspannung die dort eingestellt ist. Wie gesagt, die musst du anpassen.

**Reply by Manos1966** [reply]
> Verstehe ich das richtig, dass die Schwellwerte in % immer analog der Werte in V sein müssen?

Nein!
Die V Werte platzierst du als "Versicherung" (Fallback) fuer den Fall, dass die CANbus Kommunikation (aka praezise SOC % Werte) nicht uebermittelt.

**Comment by TheMikeCAD** [comment]
Erstmal vielen Dank!

Also es ist ein Victron 150/35.
Mir war bisher nicht klar, dass ich den einstellen kann / muss.

**Reply by Manos1966** [reply]
Haben wir alle so angefangen... (und manche, wie ich, so geblieben 😄 )

Guck aber hier:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Dynamic-Power-Limiter

**Reply by Manos1966** [reply]
Victron 150/35 Gute Wahl! 👍 

Bitte hier gucken:
https://github.com/helgeerbe/OpenDTU-OnBattery/wiki/Battery%E2%80%90Tipps

"Usefull comments about Pylontech Batteries"

**Comment by Manos1966** [comment]
ich war schneller als @spcqike 😱 
ich werde den Thread einrahmen! 😄

**Comment by TheMikeCAD** [comment]
So, Ihr habt mir echt super geholfen! Vielen Dank!
Jetzt hab ich laut https://www.victronenergy.com/live/battery_compatibility:pylontech_phantom
Die Absorbtionsspannung auf 52 V und die Erhaltungsspannung auf 51V.
die 52V entsprechen der Full-Passthrough Startspannung in der DTU.
Sind meine Werte irgendwie realistisch oder etwas zu hoch?

**Reply by knopers1** [reply]
mit 52V Absorbtionsspannung bleibt der Akku bei 90% stehen und der Laderegler geht auf Float. 
Der Akku braucht 53,2V, aber mit 52.8V geht ein US3000C ebenso auf 100%. Wenn Du die Ladespannung reduzierts, wird der Akku nicht vollständig aufgeladen. Theorie und Praxix.... Teste es bei dir aus...

**Reply by Manos1966** [reply]
> mit 52V Absorbtionsspannung bleibt der Akku bei 90% stehen und der Laderegler geht auf Float.
Der Akku braucht 53,2V, aber mit 52.8V geht ein US3000C ebenso auf 100%. 

Ich tendiere anderer Meinung, Mindestens laut der Spannungsdaten dieser Folie....
Aber korrekt gesagt... Theorie und Praxis

![Untitled](https://github.com/helgeerbe/OpenDTU-OnBattery/assets/129660360/d929495a-204a-497d-8f24-5161df80c3cf)

**Comment by knopers1** [comment]
> Erstmal vielen Dank!
> 
> Also es ist ein Victron 150/35. Mir war bisher nicht klar, dass ich den einstellen kann / muss.

den 150/35 möchte ich mir auch bald zulegen. Mit 3x400Watt in Reihe sollte das Teil einen US3000C gut laden können. Wofür ist der Potenziometer/Wahlschlter am Gerät und wie hast du das Teil eingestellt? Habe zur Zeit den 100/20 ohne den Wahlschalter verbaut.

**Comment by TheMikeCAD** [comment]
Das kann in der App gewählt werden - beim Drehschalter verbergen sich unterschiedliche Konfigurationen hinter jeder Nummer

https://www.victronenergy.com/media/pg/Manual_BlueSolar_150-35__150-45/de/configuration-and-settings.html#UUID-cb7db0b6-f183-b258-6a15-250f684ca013

Ich hab mir die Victron App geladen und damit eingestellt - dort kannst dann aussuchen, ob du den Drehschalter verwenden willst.
Den 150/35 hab ich genommen, weil der in der Victron Auslegung so berechnet wurde mit meinen Platten ;-)

**Reply by knopers1** [reply]
okay, demnach die Schalterstellung 7 für den Pylontech, wobei die Spannung sowieso noch nachgestellt werden muss. Etwas überflüssig der Schalter....
